homeassistant/components/sensor/const.py

```diff
@@ -137,6 +139,12 @@ class SensorDeviceClass(StrEnum):
     Unit of measurement: `ppm` (parts per million)
     """
 
+    CONDUCTIVITY = "conductivity"
+    """Carbon Dioxide gas concentration.
```

Comment:
Description should be Conductivity not Carbon Dioxide gas concentration.

Reply:
Sure, thnx.